src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,25 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("User can not be null");
+        }
+
+        if (user.getLogin() == null || user.getLogin().length() < 6) {
+            throw new RuntimeException("login must contain more than 6 characters");
```

Comment:
The error message should be 'login must contain at least 6 characters' to be consistent with the actual check being performed.